scripts/update-android-docs.sh

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env bash
+
+set -Eeuxo pipefail
+
+#
+# Requirements:
+#   `export GITHUB_TOKEN=...` token to access the GitHub API [https://github.com/settings/tokens].
+#   Have following tools installed:
+#     - git        - if you are a developer, you've heard about it [man 1 git].
+#     - gh         - GitHub's official command line tool [https://github.com/cli/cli].
+#     - jq         - command-line JSON processor [https://github.com/stedolan/jq].
+#     - GNU getopt - cli option parser [man 1 getopt], out-of-the-box for GNU/Linux;
+#
+
+DOKKA_OUTPUT_DIR="./build/dokka/htmlCollector"
+
+readonly ANDROID_DOCS_DIRECTORY="android-docs-repo"
+readonly CONSTANTS_FILE="./src/constants.json"
+readonly MAP_VERSION_NUMBERS_FILE="./src/data/map-version-numbers.json"
+readonly BRANCH_WITH_DOCUMENTATION="publisher-production"
+readonly ANDROID_DOCS_REPO="git@github.com:mapbox/android-docs.git"
+
+MAPS_SDK_VERSION=
+REVIEWERS=
```

Comment:
Why are those empty?

Reply:
It's initialised below from the command line inputs.

Reply:
🙇

Reply:
They will be used to save the params